configs/mmseg/segmentation_vacc-int8_static_512x512.py

```diff
@@ -0,0 +1,18 @@
+_base_ = ['./segmentation_static.py', '../_base_/backends/vacc.py']
+
+onnx_config = dict(input_shape=[512, 512])
+
+backend_config = dict(model_inputs=[
+    dict(shape=dict(input=[1, 3, 512, 512]), qconfig=dict(dtype='int8'))
+])
+
+partition_config = dict(
+    type='end2end',
+    apply_marks=True,
+    partition_cfg=[
+        dict(
+            save_file='model.onnx',
+            start=['segmentor_forward:output'],
```

Comment:
Let's say the original exported onnx is like `input->conv->...>conv->Resize->ArgMax`, here the partition setting is to skip `argmax` or both `Resize` and `argmax` ?

Reply:
I implement two methods. You can skip `Resize`  and `ArgMax` both or skip `ArgMax` only.

Reply:
In that case, we could keep the partition config and add a comment to explain which op(s) is/are skipped.